Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3
+
+ARG PATH_FOR_INITIAL_DATA="restaurants.json"
```

Comment:
`ARG` is used to specify Docker build-time argument, while the path to the data should be set in runtime with `ENV`. 

in general, `restaurants.json` should not even be added to the `Docker` image. instead, `restaurants.json` should be attached as a volume and a proper environment variable should be set during `docker run`.

Reply:
fixed